package.json

```diff
@@ -32,6 +32,10 @@
     "url": "git://github.com/ramda/ramda.git"
   },
   "main": "index.js",
+  "files": [
+    "src",
+    "dist"
+  ],
```

Comment:
It's included but weirdly [not documented](https://docs.npmjs.com/files/package.json#files)...